t/ssl_proto_version.t

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/perl
+
+use warnings;
+use Test::More;
+use FindBin qw($Bin);
+use lib "$Bin/lib";
+use MemcachedTest;
+
+if (!enabled_tls_testing()) {
+    plan skip_all => 'SSL testing is not enabled';
+    exit 0;
+}
+
+my $server = new_memcached('-o ssl_min_version=2');
+
+# Minimum supported protocol version
+$sock = $server->new_sock(undef, 'TLSv1_2');
+print $sock "version\r\n";
+like(scalar <$sock>, qr/VERSION/, "handshake with minimum proto version");
+
+# Above minimum supported protocol version
+$sock = $server->new_sock(undef, 'TLSv1_3');
```

Comment:
There's a [convention](https://github.com/memcached/memcached/blob/c472369fed5981ba8c004d426cee62d5165c47ca/t/ssl_session_resumption.t#L43) for skipping tests to ensure compatibility with environments on older versions of OpenSSL that don't ship with TLS v1.3 support, not sure if we should do the same here.

Reply:
tests shouldn't fail on older distro builds, so it should at least handle it properly.